schemas/rum/view-schema.json

```diff
@@ -360,6 +360,40 @@
             }
           },
           "readOnly": true
+        },
+        "display": {
+          "type": "object",
+          "description": "Display properties",
+          "properties": {
+            "scroll": {
+              "type": "object",
+              "description": "Scroll properties",
+              "required": ["max_depth", "max_scroll_height", "max_depth_time", "max_scroll_top"],
+              "properties": {
+                "max_depth": {
+                  "type": "number",
+                  "description": "Maximum scrolled distance on this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_scroll_height": {
+                  "type": "number",
+                  "description": "Maximum scrollable distance on this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_depth_time": {
+                  "type": "number",
+                  "description": "Timestamp of the scroll event that reached the maximum scroll depth for this view",
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏not sure how this timestamp will be used but event dates are corrected by the backend, will this one be corrected as well? if not, could it be bothering?